src/commands/wallet/get-new-address.js

```diff
@@ -0,0 +1,35 @@
+const { Command, flags: flagTypes } = require('@oclif/command');
+const getNewAddress = require('../../../lib/commands/getNewAddress');
+
+class GetNewAddressCommand extends Command {
```

Comment:
`mn core generate-to-address <amount>  --address `. Create a new wallet if address omitted